component/src/main/java/org/ballerinalang/net/jms/actions/Send.java

```diff
@@ -45,53 +40,38 @@
 import java.util.Map;
 import java.util.UUID;
 import javax.jms.Message;
-import javax.transaction.TransactionManager;
 
 import static org.ballerinalang.net.jms.Constants.EMPTY_CONNECTOR_ID;
 
 /**
- * {@code Post} is the send action implementation of the JMS Connector.
+ * {@code Send} is the send action implementation of the JMS Connector.
  */
 @BallerinaAction(packageName = "ballerina.net.jms",
                  actionName = "send",
                  connectorName = Constants.CONNECTOR_NAME,
-                 args = {
-                         @Argument(name = "jmsClientConnector",
-                                   type = TypeKind.CONNECTOR), @Argument(name = "destinationName",
-                                                                         type = TypeKind.STRING),
-                         @Argument(name = "msgType",
-                                   type = TypeKind.STRING), @Argument(name = "m",
-                                                                      type = TypeKind.MESSAGE)
+                 args = {@Argument(name = "destinationName",
+                                   type = TypeKind.STRING),
+                         @Argument(name = "message",
+                                   type = TypeKind.STRUCT)
                  },
-                 returnType = { @ReturnType(type = TypeKind.BOOLEAN) },
                  connectorArgs = {
                          @Argument(name = "properties",
-                                   type = TypeKind.MAP)
+                                   type = TypeKind.STRUCT)
                  })
 @BallerinaAnnotation(annotationName = "Description",
```

Comment:
these are not required anymore, you can add these descriptions to the native.bal file itself

Reply:
removed,
Thanks :)